bitnami/redis/templates/secret.yaml

```diff
@@ -21,3 +31,26 @@ type: Opaque
 data:
   redis-password: {{ include "redis.password" . | b64enc | quote }}
 {{- end -}}
+{{- if .Values.serviceBindings.enabled }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "common.names.fullname" . }}-svcbind
```

Comment:
How is this secret connected to the rest of the deployment? I do not see any reference in any other place.

Reply:
Basically it's a secret that is consumed by the apps through the [bindings](https://servicebinding.io/application-developer/).